msgpack/documentation/documentation.md

```diff
@@ -1,31 +1,84 @@
-# Msgpack module docummentation
+# Msgpack module documentation
 
 
-_For the moment_ we have an implementation with some many limits mainly cause by some
-restrictions of API, but we can make basic operations that can be done by the original library, like packing, unpacking and a limited use in background of msgpack objects. So
-the list of usable functions is : 
+Now, we propose two ways to use msgpack in Ark, the minimalist msgpack (with limited background object manipulation) and a more realistic msgpack module (with msgpack objects use in Ark) .
 
-#### **msgPack** : for data packing ;
-#### **msgUnpack** : for data unpacking ;
-#### **msgObjectStr** : for getting deserialized msgpack object string.
 
-## Example
+## **Minimal msgpack**
+
+
++ **msgpackMiniPack(object)** : data pack ;
++ **msgpackMiniUnpack(buffer)** : data unpack ;
++ **msgpackMiniObjectStr(buffer)** : return deserialized msgpack::object as a string .
```

Comment:
coding convention, RFC 003: `msgpack:miniPack` instead of camelCase

Also could you write the examples in ArkScript? `(msgpack:miniPack object)` instead of `msgpack:miniPack(object)`